packages/formStructure/readme/fieldGroup.md

```diff
@@ -0,0 +1,7 @@
+# FieldGroup
+
+A `FieldGroup` may be used to visually group 2 or more fields that are closely related. For example: a number input followed by a select dropdown containing possible memory units.
+
+Fields are arranged in a row, but should be switched to a column for smaller viewport widths to avoid rendering fields that are too narrow.
```

Comment:
do developers and designers need to handle this automatically or utils/ui-kit will handle?

Reply:
The field group needs to be told which breakpoint to switch at.

Reply:
OK will file JIRA for this to discuss further as seems like something UIkit can have an opinion, at least by default if doe snot have not already, so designers and dev don't have to think about and can just optimize to our laptop/desktop large viewports